scripts/install-windows-prereqs.ps1

```diff
@@ -582,17 +602,29 @@ function Install-DCAP-Dependencies {
                     Throw "Failed searching for $driver driver"
```

Comment:
Can you verify if the conditional on line 544 is still correct? It looks like the nesting is incorrect now and the scope of that very large block is only concerned with driver install, in which case it should only encompass the `if($LaunchConfiguration -eq "SGX1FLC")` configuration. This would also make the nested check on line 597 redundant.